src/databricks/labs/ucx/hive_metastore/table_migrate.py

```diff
@@ -489,3 +498,7 @@ def _sql_alter_from(self, table: Table, target_table_key: str, ws_id: int):
             f"('upgraded_from' = '{source}'"
             f" , '{table.UPGRADED_FROM_WS_PARAM}' = '{ws_id}');"
         )
+
+    def _is_migrated(self, schema: str, table: str) -> bool:
```

Comment:
Not required to implement, sharing to show my thoughts:

On one hand, I would like this method to have the following signature: `_is_migrated(self, table: Table) -> bool`. To me that makes more sense, it is "better" OO programming and it is probably more future proof (what if we want to include the catalog at some point).

On the other hand, I do not think you have to change it because: i) the `index.is_migrated` has the same signature, so it is better to be consitent now, log this as a (small) tech debt issue and do refactoring in a separate PR. (We prefer to separate refactoring from feature implementation.) and ii) it is a hidden method, therefore we can refactor the signature without worrying about changing the public API

Reply:
An issue was created. [#2397](https://github.com/databrickslabs/ucx/issues/2397)